### PR TITLE
Replace remaining metadata to follow the template

### PR DESCRIPTION
**Pull request by farshidtz**
The following have already been updated:
- edgexfoundry
- device-camera
- device-gpio

Remaining:
- [x] app-service-configurable
- [x] edgex-cli
- [x] device-grove
- [x] device-modbus
- [x] device-mqtt
- [x] device-rest
- [x] device-snmp
- [x] edgex-ui

Defer:
- device-rfid-llrp
- app-rfid-llrp-inventory

### CODE REVIEW

**Comment by farshidtz** [comment]
@siggiskulason @MonicaisHer please review while looking at the rich diff to test the hyperlinks as well. Please focus on the components you are more familiar with.

File: metadata/edgex-ui.md
Comment (MonicaisHer) [inline metadata/edgex-ui.md:7]: For a graphical user interface, see https://snapcraft.io/edgex-ui

File: metadata/edgex-ui.md
Comment (MonicaisHer) [inline metadata/edgex-ui.md:10]: Please replace with: https://github.com/edgexfoundry/edgex-ui-go/blob/main/snap/README.md

File: metadata/edgex-device-grove.md
Comment (MonicaisHer) [inline metadata/edgex-device-grove.md:11]: This URL is invalid. We dont have README for snap yet.

**Review by MonicaisHer — CHANGES_REQUESTED** [review]

File: metadata/edgex-ui.md
Comment (farshidtz) [reply metadata/edgex-ui.md:7]: This is the UI's description, redirecting users the CLI is that's what they prefer.

**Review by MonicaisHer — APPROVED** [review]
LGTM. Thanks a lot. @farshidtz

File: metadata/edgex-device-mqtt.md
Comment (siggiskulason) [inline metadata/edgex-device-mqtt.md:3]: Remove "The"

File: metadata/edgex-device-snmp.md
Comment (siggiskulason) [inline metadata/edgex-device-snmp.md:3]: Remove "The"

File: metadata/edgex-device-camera.md
Comment (farshidtz) [reply metadata/edgex-device-camera.md:9]: Yes that would be good. I'd consider that for all protocols in descriptions in future updates.

File: metadata/edgex-device-grove.md
Comment (farshidtz) [reply metadata/edgex-device-grove.md:9]: I think the readme indicates support for 3 and higher by writing 3+. I took that by mistake thinking that it refers to Raspberry Pi 3 A+/B+ models. 

https://github.com/edgexfoundry/device-grove-c#supported-boards

Comment (farshidtz) [reply metadata/edgex-device-grove.md:9]: Perhaps it is even supported on the new Raspberry Pi Zero 2 W, which has a 64-bit processor.

Comment (farshidtz) [reply metadata/edgex-device-grove.md:9]: How about:
> This device service is expected to work on the following boards:
> * Raspberry Pi 3 or higher - arm64 OS
> * Raspberry Pi Zero 2 or higher - arm64 OS

File: metadata/edgex-device-modbus.md
Comment (farshidtz) [reply metadata/edgex-device-modbus.md]: I replaced but also mentioned the read/write capability.

File: metadata/edgex-cli.md
Comment (farshidtz) [reply metadata/edgex-cli.md:4]: Changed to
> EdgeX CLI is a command line interface tool for developers, used for interacting with the EdgeX microservices.

File: metadata/edgex-device-camera.md
Comment (farshidtz) [reply metadata/edgex-device-camera.md]: It is important to say the service adds something to EdgeX, rather than being a standalone component created by EdgeX.

I replaced with:
> The EdgeX Device Camera is a device service for communicating with ONVIF-compliant cameras via EdgeX.
It can be used to get and set camera attributes and retrieve snapshot images.

I think including what is supported in sufficient.

File: metadata/edgex-device-grove.md
Comment (siggiskulason) [reply metadata/edgex-device-grove.md:9]: > 

that sounds good

**Review by siggiskulason — APPROVED** [review]
LGTM

**Comment by siggiskulason** [comment]
Note that this repository is not owned by the EdgeX Team, so Mengyi and I don't have write access - and can't formally approve the PR.

**Comment by farshidtz** [comment]
Thanks a lot @siggiskulason @MonicaisHer.
I went ahead and updated the metadata on the store. Will merge this PR once given the rights.

**Pull request merged**
